doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -93,6 +93,10 @@ Please make sure to set your preferred storage location ***before*** migration.
   - Removed commands: `pimcore:bundle:enable`, `pimcore:bundle:disable`.
   - Removed `dontCheckEnabled` config support from Areablock editable.
 - [Codeception] Bumped `codeception/codeception` version to ^5.0. Now, Pimcore is using a new directory structure for tests (Codeception 5 directory structure). For details, please see [#13415](https://github.com/pimcore/pimcore/pull/13415)
+- [Session] 
+  - `AdminSessionHandler` requires session from request stack.
+  - `EcommerceFrameworkBundle\SessionEnvironment` does not work in cli mode anymore.
```

Comment:
you still can use `SessionEnvironment` in cli mode, but it does not load/store any values from/to session - which does not make any sense in cli mode anyway. 
right? 
so 'does not work' is probably not right?

Reply:
I meant it does nothing in cli mode as we return the control from every method, if called from cli. I can change to 'not supported'.

Reply:
not supported is also not entirely correct, since you still can use it in cli mode too ... I changed the phrase a bit.